docs/resources/zk.md

```diff
@@ -0,0 +1,89 @@
+---
+page:
+  headHtml: |
+    <snippet var="js.highlightjs" />
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.5.1/languages/ini.min.js"></script>
```

Comment:
Imagine if we could just `<snippet var="js.highlightjs-lang.ini" />`. 

(Don't have to do this in this PR.)

Reply:
Even better if the snippets could accept arbitrary variables, like

```html
<snippet var="js.highlightjs-lang" lang="ini" />

<!-- uses: -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.5.1/languages/{{ lang }}.min.js"></script>

<!-- results in: -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.5.1/languages/ini.min.js"></script>
```